packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -9,12 +9,17 @@ export const containerClass = style([
     display: 'flex',
     flexDirection: 'row',
     flexWrap: 'nowrap',
+    width: 'max-content',
```

Comment:
I believe you can do this 
```suggestion
    width: {
      xs: "max-content",
      sm: "100%",
    }
```